### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by JamesKunstle**
had to make small pandas API changes since we're updating like 6 versions.

### CODE REVIEW

**Comment by JamesKunstle** [comment]
I've unpinned all the versions to catch everything up, but I'm going to repin them after this PR. I'm going to try to use pipenv but if that still sucks I'll check out Poetry

File: 8Knot/pages/contributions/visualizations/cntrb_pr_assignment.py [hunk 1]
```diff
@@ -220,7 +222,7 @@ def process_data(df: pd.DataFrame, interval, assign_req, start_date, end_date):
         .value_counts()
         .to_frame()
         .reset_index()
-        .rename(columns={"assignee": "count", "index": "assignee"})
+        # .rename(columns={"assignee": "count", "index": "assignee"})
```
Comment (cdolfi) [inline 8Knot/pages/contributions/visualizations/cntrb_pr_assignment.py:225]: Should this comment still be there?

File: 8Knot/pages/contributions/visualizations/cntrib_issue_assignment.py [hunk 2]
```diff
@@ -217,7 +219,7 @@ def process_data(df: pd.DataFrame, interval, assign_req, start_date, end_date):
         .value_counts()
         .to_frame()
         .reset_index()
-        .rename(columns={"assignee": "count", "index": "assignee"})
+        # .rename(columns={"assignee": "count", "index": "assignee"})
```
Comment (cdolfi) [inline 8Knot/pages/contributions/visualizations/cntrib_issue_assignment.py:222]: same as above

**Review by cdolfi — CHANGES_REQUESTED** [review]
Small changes and needs to be brought up to date. Once that is done and an issue is created for repinning itll be good to merge!

File: 8Knot/pages/chaoss/visualizations/contrib_importance_pie.py [hunk 3]
```diff
@@ -318,7 +318,12 @@ def process_data(df: pd.DataFrame, action_type, top_k, patterns, start_date, end
     df_sum = df[action_type].sum()
 
     # calculate the remaining contributions by taking the the difference of t_sum and df_sum
-    df = df.append({"cntrb_id": "Other", action_type: t_sum - df_sum}, ignore_index=True)
+    # df = df.append({"cntrb_id": "Other", action_type: t_sum - df_sum}, ignore_index=True)
```
[the remark this answers is not in the document]
Comment (JamesKunstle) [reply 8Knot/pages/chaoss/visualizations/contrib_importance_pie.py:321]: removed

File: 8Knot/pages/contributors/visualizations/contrib_importance_pie.py [hunk 4]
```diff
@@ -319,7 +319,12 @@ def process_data(df: pd.DataFrame, action_type, top_k, patterns, start_date, end
     df_sum = df[action_type].sum()
 
     # calculate the remaining contributions by taking the the difference of t_sum and df_sum
-    df = df.append({"cntrb_id": "Other", action_type: t_sum - df_sum}, ignore_index=True)
+    # df = df.append({"cntrb_id": "Other", action_type: t_sum - df_sum}, ignore_index=True)
```
[the remark this answers is not in the document]
Comment (JamesKunstle) [reply 8Knot/pages/contributors/visualizations/contrib_importance_pie.py:322]: removed

**Comment by JamesKunstle** [comment]
@cdolfi 
1) re-pinned dependencies using modern Python app dependency management best-practice
2) removed comments you notes
3) I think the commits I made are discrete changes and shouldn't be squashed.

**Comment by cdolfi** [comment]
@JamesKunstle Sweet, can you bring the branch back up to date and ill merge! Not too sure how it got out of date but since it is package stuff i want to make sure we are thorough

**Comment by JamesKunstle** [comment]
@cdolfi should be up to date

**Comment by JamesKunstle** [comment]
@cali squashed commits, rebased onto current dev,  should be good to go

**Review by cdolfi — APPROVED** [review]
lgtm and ran on fresh build

**Pull request merged**